csuntechlab/iSTART
# STR-34 -- BE: Setup pairing between student id (Members:) and provided research_id

# Description
'Write a meaningful description of your task'
  Create a table that contains the pairing between user_id and research_id.
  Login now checks against this table to see if a non-admin user is allowed into the application
# Testing
'What does the reviewer need to do to test your PR?'
  1. Be very strict on my back-end, variables, and what I am returning.
  2. Login as `april.feldman@csun.edu` and then:
    - open the console log
    - click on the `network` tab
    - click on response to see:      
       `"research_id":true`
   - This assures that the user has a `research_id`
  3. Then login as someone who has a `research_id` such as `rachel.knopf@csun.edu` but has not logged in yet
    - open the console log
    - click on the `network` tab
    - click on response to see:      
       `"research_id":true`
   - This assures that the user has a `user_group` and a `research_id` of 1
4. Then login as someone who hasn't logged in before as `jamison.keller@csun.edu`, `victoria.branch@csun.edu`
    - open the console log
    - click on the `network` tab
    - click on response to see:      
       `valid => 0`
   - This assures that the user doesn't have a `research_id` and can not log in
 
# Installation
'Are there any dependencies the reviewer needs to download?'
You need to run `php artisan migrate:fresh --seed`
